src/components/interaction-designer/block-editors/GenericContactPropertyEditor.vue

```diff
@@ -232,6 +232,13 @@ export class GenericContactPropertyEditor extends mixins(Lang) {
     return !isEmpty(this.subscriberPropertyFields)
   }
 
+  get subscriberPropertyFieldsForSelector() {
+    return map(this.subscriberPropertyFields, (field: IContactPropertyOption) => {
+      const isFieldTypeMatchedWithCurrentBlock = this.subscriberPropertyFieldDataTypesMapping[field.data_type].includes(this.block.type)
+      return {...field, $isDisabled: isFieldTypeMatchedWithCurrentBlock === false}
```

Comment:
why not `!isFieldTypeMatchedWithCurrentBlock`?

Reply:
Good point, just changed!